docs/usage.md

```diff
@@ -122,6 +126,7 @@ contour-plus interprets following annotations for HTTPProxy.
 - `cert-manager.io/issuer` - The name of an  [Issuer][] to acquire the certificate required for this HTTPProxy from. The Issuer must be in the same namespace as the HTTPProxy.
 - `cert-manager.io/cluster-issuer` - The name of a [ClusterIssuer][Issuer] to acquire the certificate required for this ingress from. It does not matter which namespace your Ingress resides, as ClusterIssuers are non-namespaced resources.
 - `kubernetes.io/tls-acme: "true"` - With this, contour-plus generates Certificate automatically from HTTPProxy.
+- `contour-plus.cybozu.com/delegated-domain: "acme.example.com"` - With this, contour-plus generates will create a CNAME record pointing to the delegation domain for use when performing DNS-01 DCV during the Certificate creation.
```

Comment:
```suggestion
- `contour-plus.cybozu.com/delegated-domain: "acme.example.com"` - With this, contour-plus will create a CNAME record pointing to the delegation domain for use when performing DNS-01 DCV during the Certificate creation.
```

Reply:
How about noting that Issuer/ClusterIssuer needs `cnameStrategy: Follow` setting and the credentials for the delegated zone?

Reply:
Granted, to use a delegated domain the Issuer and domain must be properly configured, but mirroring the rest of the document, configuration of external components is left as an exercise to the reader. An administrator looking to use this feature is most likely to be already familiar with its setup.